training/src/anemoi/training/commands/mlflow.py

```diff
@@ -8,9 +8,24 @@
 # nor does it submit to any jurisdiction.
 
 
+from __future__ import annotations
+
 import argparse
+import getpass
+import json
+import logging
+import tempfile
+from pathlib import Path
+
+import mlflow
+from hydra import compose
+from hydra import initialize
+from omegaconf import OmegaConf
 
 from anemoi.training.commands import Command
+from anemoi.training.diagnostics.mlflow.client import AnemoiMlflowClient
```

Comment:
Please put all imports that are not in the standard library in the subcommand, so they are only imported when the command is executed.